.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Talk with Us on Slack
```

super nit: capitalization
```suggestion
  - name: Talk with us on Slack
```